config/configuration/bundles/src/bundle/org/sakaiproject/config/bundle/default.sakai.properties

```diff
@@ -1913,6 +1913,10 @@
 # Default: false
 # gradebook.coursegrade.displayed=true
 
+# Default for course grade calculated
+# Default: true
+# gradebook.coursegrade.isCalculatedGrade=true
```

Comment:
This should be set to false

Reply:
Thanks, @jesusmmp

Reply:
gradebook.coursegrade.showCalculatedGrade